velox/connectors/Connector.h

```diff
@@ -163,6 +163,8 @@ class ConnectorInsertTableHandle : public ISerializable {
 
   virtual std::string toString() const = 0;
 
+  virtual const std::vector<TypePtr> getColumnHandleDataTypes() const = 0;
```

Comment:
I am thinking if we should add this API to the TableWriteNode. This interface seems to have minimal API calls so far.